src/services/review/ReviewRequestService.ts

```diff
@@ -224,9 +224,13 @@ export default class ReviewRequestService {
     const { siteName, isomerUserId: userId } = sessionData
 
     // Find all review requests associated with the site
-    const requests = await this.repository.findAll({
+    // TODO: Note this needs to be findAll when we reach a stage of allowing
```

Comment:
maybe we should file an issue for awareness?

Reply:
IS-215